kolena/workflow/test_run.py

```diff
@@ -401,14 +407,44 @@ def _iter_test_samples_batch(
             df_class=TestSampleDataFrame,
         )
 
+    def _extract_thresholded_metrics(
+        self,
+        records: List[Tuple[Dict[str, Any], Dict[str, Any]]],
+    ) -> Tuple[List[Tuple[Dict[str, Any], Dict[str, Any]]], List[Tuple[Dict[str, Any], List[Dict[str, Any]]]]]:
+        updated_records = []
+        removed_items = []
```

Comment:
```suggestion
        standard_metrics = []
        thresholded_metrics = []
```

nit: It's really confusing to read this and wonder what `removed_items` is, especially given that the method is called `_extract_thresholded_metrics` -- suggest updating in this method + in the unit tests

Reply:
updated with this suggestions